diff --git a/src/App.h b/src/App.h
index 27f10b3..179b57b 100644
--- a/src/App.h
+++ b/src/App.h
@@ -231,12 +231,6 @@ public:
 
     TemplatedApp(SocketContextOptions options = {}) {
         httpContext = HttpContext<SSL>::create(Loop::get(), options);
-
-        /* Register default handler for 404 (can be overridden by user) */
-        this->any("/*", [](auto *res, auto */*req*/) {
-            res->writeStatus("404 File Not Found");
-            res->end("<html><body><h1>File Not Found</h1><hr><i>uWebSockets/20 Server</i></body></html>");
-        });
     }
 
     TemplatedApp& operator=(const TemplatedApp&) = delete;
